react/Geolocation.js

```diff
@@ -129,32 +161,31 @@ class Geolocation extends Component {
         this.setState({ permissionStatus: DENIED })
         searchPickupAddressByGeolocationEvent({
           deniedGeolocation: true,
+          elapsedTime,
         })
         break
       case 2: // POSITION_UNAVAILABLE
         setAskForGeolocation(false)
         setGeolocationStatus(ERROR_NOT_FOUND)
-        this.setCurrentActiveState(ERROR_NOT_FOUND)
+        this.setCurrentActiveState(INITIAL)
         searchPickupAddressByGeolocationEvent({
           confirmedGeolocation: true,
           positionUnavailable: true,
+          elapsedTime,
         })
         break
       case 3: // TIMEOUT
-        // TODO#2: look into retrying timeout, refer to TODO#1
```

Comment:
Looked into this, retrying this particular function (`getCurrentPosition`) has the same effect as just increasing the timeout.